Makefile

```diff
@@ -50,7 +52,7 @@ check-kubo-rpc-server:
 
 .PHONY: release
 release:
-	RUSTFLAGS="-D warnings" cargo build -p ceramic-one --locked --release
+	RUSTFLAGS="-D warnings" $(CARGO) build -p ceramic-one --locked --release
```

Comment:
the `RUSTFLAGS` is redefined in the `$(CARGO)`

should `CARGO = RUSTFLAGS='--cfg tokio_unstable' cargo`
be more like `CARGO = RUSTFLAGS=$RUSTFLAGS + '--cfg tokio_unstable' cargo`

Reply:
Updated the Makefile to handle appending to RUSTFLAGS. Thanks for catching that